client/app/pages/queries/schema-browser.js

```diff
@@ -3,6 +3,9 @@ import template from './schema-browser.html';
 function SchemaBrowserCtrl($scope) {
   'ngInject';
 
+  this.versionToggle = false;
+  this.versionFilter = 'abcdefghijklmnop';
```

Comment:
Because I can't put an if on the additional filter for listing the tables, I needed something that was highly unlikely to be a table name. Leaving it as an empty string provides a case where no tables are listed.